openai/openai-vanilla/runtime/src/main/java/io/quarkiverse/langchain4j/openai/runtime/config/ChatModelConfig.java

```diff
@@ -68,4 +68,11 @@ public interface ChatModelConfig {
      */
     @ConfigDocDefault("false")
     Optional<Boolean> logResponses();
+
+    /**
+     * The response format the model should use.
+     * Some models are not compatible with some response formats, make sure to review OpenAI documentation.
+     */
+    @WithDefault("text")
```

Comment:
Shouldn't we leave this unset if the user does not select something?

It seems to me that the default in LangChain4j is to not set a value

Reply:
Indeed, I went with the default as indicated by OpenAI doc, LangChain4j doesn't set anything.
How do you handle this scenario? `Optional<String>` then pass the property with something like `chatModelConfig.responseFormat().orElse(null)`? Or is there another pattern I'm not aware of?

Reply:
Yeah, exactly that